treemacs-branch-creation.el

```diff
@@ -178,9 +178,14 @@ correct cache entries."
           (button-put b 'abs-path (nth (- (length it) 1) it))
           (button-put b 'parent-path root)
           (end-of-line)
-          (let ((beg (point)))
-            (insert (cadr it))
-            (add-text-properties beg (point) props)))))))
+          (let* ((beg (point))
+		 (dir (cadr it))
+		 (parent (file-name-directory dir)))
+	    (insert dir)
+	    (add-text-properties beg (point) props)
+	    (add-text-properties
+	     (+ (line-beginning-position) 2) (+ beg (length parent))
```

Comment:
The start position must be `(button-start b)`, otherwise you'll also visible fontify the indentation when `treemacs-indentation-string` is something other than whitespace.

Reply:
Fixed.